command/ca/init.go

```diff
@@ -845,3 +849,14 @@ func normalize(name string) string {
 	}
 	return name
 }
+
+// allEmpty loops through all strings in the slice and returns if
+// all are empty (length 0).
+func allEmpty(parts []string) bool {
+	for _, p := range parts {
+		if len(p) > 0 {
```

Comment:
Is their a reason we're specifically checking for `len` here? With string in golang I thought it was more idiomatic to check if it's equivalent to `""`.

Reply:
Fixed in 3dca2ff78a529a346abffaf41075be45a21b8104. You're right; it's more idiomatic that way.